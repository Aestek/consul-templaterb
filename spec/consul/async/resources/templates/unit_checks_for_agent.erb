Consul Agent Info
<%
  consul_info = agent_self()
  consul_metrics = agent_metrics()
%>-------
NodeName: <%= consul_info['Config']['NodeName'] %>
Consul Metrics
-------
<%
consul_metrics['Samples'].each do|metric|
  %><%= metric['Name']%> Mean: <%= metric['Mean']%>
<% end %>