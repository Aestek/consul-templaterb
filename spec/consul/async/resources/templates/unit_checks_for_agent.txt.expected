Consul Agent Info
-------
NodeName: C02Q3L7GG8WP
Consul Metrics
-------
consul.consul.fsm.coordinate.batch-update Mean: 0.06999299675226212
consul.memberlist.gossip Mean: 0.014945925005013122
consul.raft.commitTime Mean: 0.024396000429987907
consul.raft.fsm.apply Mean: 0.10652600228786469
consul.raft.leader.dispatchLog Mean: 0.011900999583303928
consul.serf.queue.Event Mean: 0.5
consul.serf.queue.Intent Mean: 0
consul.serf.queue.Query Mean: 0
